example/package.json

```diff
@@ -9,6 +9,8 @@
   "license": "MIT",
   "dependencies": {
     "core-js": "^3.6.4",
+    "eslint-plugin-jest": "^23.8.2",
+    "jest": "^25.3.0",
```

Comment:
These should be devDependencies. Not dependencies. They are only used for development and not needed at runtime.

Reply:
Fixed